Makefile

```diff
@@ -36,6 +35,15 @@ PKG_MANAGER ?= $(shell command -v dnf yum|head -n1)
 # ~/.local/bin is not in PATH on all systems
 PRE_COMMIT = $(shell command -v bin/venv/bin/pre-commit ~/.local/bin/pre-commit pre-commit | head -n1)
 
+COVERAGE_PATH ?= $(shell pwd)/.coverage
+COVERAGE_PROFILE ?= $(shell pwd)/coverage.txt
+export COVERAGE_PATH
+export COVERAGE_PROFILE
+$(shell mkdir -p ${COVERAGE_PATH})
+
+CODECOVE_TOKEN=52d2da88-a97c-442b-8c9f-1f77a5a26b7c
```

Comment:
Yikes. Shouldn't this be a Cirrus secret?

Reply:
Absolutely. I can remove it now and generate a new token if you want to. I thought it's fine for testing purposes.